.github/actions/height-integration/sync-pr/pr-labels-updated-action.yml

```diff
@@ -0,0 +1,24 @@
+name: Sync updated PR labels to Height
+description: Moves any issue-related tasks in Height into the relevent swimlanes on the interface project board
```

Comment:
not sure if this one moves into swimlane - we should creates tags
![Screenshot 2023-01-06 at 16 24 17](https://user-images.githubusercontent.com/8342048/211053943-3e55918b-a9d0-43c5-a889-781440a64e68.png)

Reply:
![Screenshot 2023-01-06 at 16 24 20](https://user-images.githubusercontent.com/8342048/211054036-8f4e7a93-fd22-4d41-b8be-784b671098b4.png)